terraform/modules/data-share-service/lb-security.tf

```diff
@@ -38,6 +38,23 @@ resource "aws_security_group" "lb_auto" {
   }
 }
 
+resource "aws_security_group" "lb_test" {
+  name_prefix = "${var.environment}-ecs-lb-test"
+  description = "Access to LB test port"
+  vpc_id      = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 8080
```

Comment:
Is there any risk here by exposing port 8080? Does that mean that clients can potentially reach this/a not-quite-production environment?

Reply:
Good point, I will look at limiting what can access this